package.json

```diff
@@ -49,6 +50,8 @@
     "eslint-plugin-jsx-a11y": "^6.7.1",
     "eslint-plugin-react": "^7.32.2",
     "eslint-plugin-react-hooks": "^4.6.0",
+    "react-redux": "^8.0.5",
```

Comment:
- [ ] We need `react-redux` package for our app to work in production, which means we should put it in `dependencies` instead of `devDependencies`. Kindly install it as a normal dependency. 🤓️